toolkit/resources/manifests/package/toolchain_aarch64.txt

```diff
@@ -352,13 +352,13 @@ sqlite-devel-3.32.3-1.cm1.aarch64.rpm
 sqlite-libs-3.32.3-1.cm1.aarch64.rpm
 swig-3.0.12-4.cm1.aarch64.rpm
 swig-debuginfo-3.0.12-4.cm1.aarch64.rpm
-systemd-239-31.cm1.aarch64.rpm
+systemd-239-32.cm1.aarch64.rpm
 systemd-bootstrap-239-29.cm1.aarch64.rpm
```

Comment:
Do these changes need to be applied to the bootstrap version as well?

Reply:
AFAIK, not all changes needed in bootstarp. I think this change is also not needed. It is to add additional flags for portablectl.